java-azure-functions/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Rookout + Azure Functions
+
+## Quickstart
+
+1. Set up the Azure Functions CLI
+
+2. Deploy the code:
+
+```
+mvn clean package azure-functions:deploy
+```
+
+3. Set Rookout token either via the Azure Portal or the CLI:
+```
+az functionapp config appsettings set --name jvmazure-20200323170137626 --settings  "ROOKOUT_TOKEN=<token>" --resource-group java-functions-group
+```
+
+4. Run the function
```

Comment:
Is this working consistently? Didn't we see app settings / env vars are inconsistent? Or is this only the JAVA_OPTS and the command line args setting?

Reply:
The env vars are consistent, the JVM command line wasn't